Stripe/STPAPIClient+Private.h

```diff
@@ -11,21 +11,26 @@
 #import "STPAPIClient.h"
 #import "STPAPIRequest.h"
 
-NS_ASSUME_NONNULL_BEGIN
-
 @class STPEphemeralKey;
 
-@interface STPAPIClient()
+NS_ASSUME_NONNULL_BEGIN
+
+@interface STPAPIClient ()
 
 + (NSString *)apiVersion;
 
 - (void)createTokenWithParameters:(NSDictionary *)parameters
                        completion:(STPTokenCompletionBlock)completion;
 
-- (NSURLSessionDataTask *)retrieveSourceWithId:(NSString *)identifier clientSecret:(NSString *)secret responseCompletion:(STPAPIResponseBlock)completion;
 
-@property (nonatomic, readwrite) NSURL *apiURL;
-@property (nonatomic, readwrite) NSURLSession *urlSession;
+@property (nonatomic, strong, readwrite) NSURL *apiURL;
+@property (nonatomic, strong, readwrite) NSURLSession *urlSession;
+
+@end
+
+@interface STPAPIClient (SourcesPrivate)
```

Comment:
We cant have both `STPAPIClient (Sources)` in both the public and private header so this is a workaround for that. What do you think?